R/rmd_to_md.R

```diff
@@ -11,22 +11,31 @@
 #' @param fig_url_dir URL path that will be used to link to the figures in the
 #'   markdown output.
 #' @param order Order of the article in the menu.
+#' @param title Title of the article, to show in the menu.
```

Comment:
Does this set:

- `title`: what appears at the top of the page
- or `sidebar.label`: what appears in the navigation on the left?

If the latter, then I would:

- rename to `sidebar_label`
- rename `order` to `sidebar_order`
- Switch params, so `sidebar_label` is first (cf. [Stardocs](https://starlight.astro.build/reference/frontmatter/#sidebarconfig) doc)